views/ABViewPageCore.js

```diff
@@ -50,7 +50,8 @@ module.exports = class ABViewPageCore extends ABViewContainer {
       //		translations:[]
       // 	}
 
-      this.parent = null; // will be set by the pageNew() that creates this obj.
+      this.parent = parent;
+      // this.parent = null; // will be set by the pageNew() that creates this obj.
```

Comment:
My biggest concern on this whole set of changes is this line right here. 

The comments right below this change tells us that setting ```this.parent = null;``` is intentional and should be done this way.

We've just broken that assumption.  Now it's been years since we've had this change in place and I don't think any of us remembers why that assumption was there or why it mattered.  And worse yet, we don't have a set of test suites that we can run to make sure the changes you are making here properly cover what needs to change.

So ... without those tests we could run to make sure the original assumptions have been properly modified ... I feel like we are just shooting in the Dark here.